core/src/test/java/dev/streamx/cli/ingestion/IngestionArgumentsValidationTest.java

```diff
@@ -37,14 +37,7 @@ public void shouldRejectChannellessIngestion(QuarkusMainLauncher launcher) {
         Error: Missing required argument(s): (<channel> <key>)
         Usage: streamx unpublish [-hV] [[--ingestion-url=<propagateIngestionUrl>]]
                                  (<channel> <key>)
-        Send unpublication trigger
-              <channel>   Channel that message will be published to
-              <key>       Message key
-          -h, --help      Show this help message and exit.
-              --ingestion-url=<propagateIngestionUrl>
-                          Address of 'rest-ingestion-service'
-                            Default: http://localhost:8080
-          -V, --version   Print version information and exit.""");
+        Try 'streamx unpublish --help' for more information.""");
```

Comment:
Here we have it, shorter command line boilerplate, especially in the below test for publish command

Reply:
I also verified, that the only remaining long command line errors, are:
1. `shouldRejectInvalidFile`
```
          Payload could not be parsed.
                    
          Supplied payload:
          asdf{][
                    
          Make sure that:
           * it's valid JSON,
           * object property names are properly single-quoted (') or double-quoted ("),
           * strings are properly single-quoted (') or double-quoted (")
                    
          Details: Unrecognized token 'asdf': was expecting (JSON String, Number, Array, Object
           or token 'null', 'true' or 'false')
           at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled);
           line: 1, column: 6]
```

2. `shouldRejectInvalidDataJson`
```
Payload could not be parsed.
                    
          Supplied payload:
          asdf{][
                    
          Make sure that:
           * it's valid JSON,
           * object property names are properly single-quoted (') or double-quoted ("),
           * strings are properly single-quoted (') or double-quoted (")
                    
          Details: Unrecognized token 'asdf': was expecting (JSON String, Number, Array, Object
           or token 'null', 'true' or 'false')
           at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled);
           line: 1, column: 6]
```

Those seem OK to remain long/descriptive